src/components/buttons/pages/buttons-interactive.jsx

```diff
@@ -9,30 +8,7 @@ const Buttons = () => {
   const allIcons = {};
 
   Object.keys(ICON_TYPES).forEach(type =>
-    allIcons[type] = <Icon type={ICON_TYPES[type]} color={ICON_COLOR.ADAPTIVE} size={16} />);
-
-  const primarySettings = [
-    {
-      name: 'buttonType',
-      values: BUTTON_PRIMARY_TYPE
-    },
-    {
-      name: 'wide',
-      values: Boolean
-    },
-    {
-      name: 'disabled',
-      values: Boolean
-    },
-    {
-      name: 'icon',
-      values: allIcons
-    },
-    {
-      name: 'href',
-      values: String
-    }
-  ];
+    allIcons[type] = <Icon type={ICON_TYPES[type]} color={ICON_COLOR.ADAPTIVE} size={24} />);
```

Comment:
have you considered using sth like this ?
```js
const allIcons = Object.entries(ICON_TYPES).reduce((acc, [key, type]) => ({
  ...acc, 
  [key]: <Icon type={type} color={ICON_COLOR.ADAPTIVE} size={24} />
})
, {});
```

Reply:
@mpacholec yeah, seems like a better solution, I think. Thx.